pysb/tests/test_importers.py

```diff
@@ -0,0 +1,129 @@
+import pysb
+import os
+from pysb.bng import BngConsole
+from pysb.importers.bngl import model_from_bngl, BnglImportError
+from pysb.importers.sbml import model_from_sbml
+import numpy
+from nose.tools import assert_raises_regexp
+
+
+def bngl_import_compare_simulations(bng_file, sim_times=range(0, 100, 10)):
+    """
+    Test BNGL file import by running an ODE simulation on the imported model
+    and on the BNGL file directly to compare trajectories.
+    """
+    m = model_from_bngl(bng_file)
+
+    # Simulate using the BNGL file directly
+    with BngConsole(model=None, suppress_warnings=True) as bng:
+        bng.load_bngl(bng_file)
+        bng.generate_network()
+        bng.action('simulate', method='ode', sample_times=sim_times)
+        yfull1 = bng.read_simulation_results()
+
+    # Convert to a PySB model, then simulate using BNG
+    with BngConsole(m, suppress_warnings=True) as bng:
+        bng.generate_network()
+        bng.action('simulate', method='ode', sample_times=sim_times)
+        yfull2 = bng.read_simulation_results()
+
+    # Check all species trajectories are equal (within numerical tolerance)
+    for species in m.species:
+        print(species)
+        print(yfull1[species])
+        print(yfull2[species])
+        print(numpy.allclose(yfull1[species], yfull2[species]))
+        assert numpy.allclose(yfull1[species], yfull2[species])
```

Comment:
Given that this should be the exact same set of ODEs, the default `allclose` tolerance might be too loose! Maybe we don't want to depend on the BNG ODE simulation being deterministic and expect precise equality, but we should at least make the tolerance here as tight as the default BNG solver's tolerance.